scripts/prepare/config.ts

```diff
@@ -64,4 +65,8 @@ export const destination = {
     new URL('packages/java/tests/spring/react-signals/src/main/resources/vaadin-core-versions.json', root),
   ],
   themeDir: new URL('packages/java/hilla/src/main/java/com/vaadin/hilla/theme/', root),
+  components: new URL(
```

Comment:
Do we use this section anywhere? 🤔

Reply:
Yes, it's the destination for the generated files.